src/renderer/src/routes/query-builder/components/DatabaseSelector.tsx

```
@@ -1,23 +1,47 @@
+
+  const loadTables = () => {
+    window.api
+      .listAvailableTables({ region: activeAWSRegion, accountId, roleName })
+      .then(setTables)
+      .catch(console.error);
+  };
+
+  /**
+   * Only run on first render
+   */
   useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loadTables

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)